### PR TITLE
[QACI-624 QACI-543] archiving server logs

### PR DESCRIPTION
**Pull request by lprimak**
## Description
This is for the PR testing

- removed rerun tests band-aids
- archiving server logs for all tests

### CODE REVIEW

**Comment by lprimak** [comment]
jenkins test

**Comment by lprimak** [comment]
jenkins test

**Comment by lprimak** [comment]
jenkins test

**Comment by lprimak** [comment]
jenkins test

**Comment by lprimak** [comment]
jenkins test

**Comment by lprimak** [comment]
jenkins test

**Comment by lprimak** [comment]
jenkins test

**Comment by lprimak** [comment]
jenkins test

File: Jenkinsfile [hunk 1]
```diff
@@ -102,19 +113,45 @@ pipeline {
...
+                my_string_property='haha'
+                MY_STRING_PROPERTY='woohoo'
+            }
             steps {
                 echo '*#*#*#*#*#*#*#*#*#*#*#*#  Running test  *#*#*#*#*#*#*#*#*#*#*#*#*#*#*#'
                 sh """mvn -B -V -ff -e clean verify --strict-checksums \
                 -Djavax.net.ssl.trustStore=${env.JAVA_HOME}/jre/lib/security/cacerts \
                 -Djavax.xml.accessExternalSchema=all -Dpayara.version=${pom.version} \
                 -Dpayara_domain=${DOMAIN_NAME} -Duse.cnHost=true \
-                -Dsurefire.rerunFailingTestsCount=2 -Ppayara-server-managed,payara5"""
```
Comment (dmatej) [inline Jenkinsfile:112]: :+1:

**Comment by lprimak** [comment]
jenkins test

**Comment by lprimak** [comment]
Jenkins test

**Comment by lprimak** [comment]
jenkins test

**Comment by lprimak** [comment]
jenkins test

**Comment by lprimak** [comment]
jenkins test

**Review by dmatej — APPROVED** [review]

**Comment by lprimak** [comment]
jenkins test

**Comment by lprimak** [comment]
jenkins test

**Comment by lprimak** [comment]
Thanks @dmatej I finally have a chance to work / complete this :)

**Comment by lprimak** [comment]
jenkins test

**Pull request merged**